js/msp/MSPHelper.js

```diff
@@ -0,0 +1,1811 @@
+'use strict';
+
+
+function MspHelper () {
+    MspHelper.prototype.process_data = function(dataHandler) {
```

Comment:
Ok fixed this. Haven't really worked with "oop" for javascript earlier and still learning. What is the benefit of defining it likes this? Seems a bit more confusing then defining the methods within the class declaration as in other languages.